<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversor de VÃ­deo (AVI e MP4) para GIF</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 197, 197);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>33</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>Origem:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_ad_origem">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>24</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color: rgb(85, 85, 255);
  color: rgb(255, 255, 255);
  border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_origem">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>651</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_converter">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>77</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 209, 122);
  color: rgb(255, 255, 255);
  border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Converter</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_status">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>921</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 85, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
